relayer/relayer/src/redis/mod.rs

```diff
@@ -110,7 +110,7 @@ impl RedisManager {
                         name.clone(),
                         ETH_BLOCK_KEY.to_string(),
                         Arc::clone(&redis_connection),
-                        seal_block_number
+                        seal_block_number + 1
```

Comment:
question is what do we want this value to be - last processed block or the next block to process. Next block makes sense because that is how the relayer works rn, but conceptually storing last processed block makes sense as well - for manual inspection, or if some other software needs to read it.  Small price to pay is if relayer crashed he goes through that block again, but it's harmless

Reply:
From how the code uses it - it should be next block to process and I think it's ok to keep it that way